src/app/libs/notification/notifications.module.ts

```diff
@@ -0,0 +1,30 @@
+import { HTTP_INTERCEPTORS } from '@angular/common/http';
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { MAT_SNACK_BAR_DEFAULT_OPTIONS } from '@angular/material/snack-bar';
+import { ErrorInterceptor } from '@core/interceptors/error.interceptor';
+import { NotificationComponent } from './components/notification.component';
+
+@NgModule({})
+export class NotificationsModule {
+
+    static forRoot (): ModuleWithProviders<NotificationsModule> {
+
+        return {
+            ngModule: NotificationsModule,
+            providers: [
+                {
+                    multi: true,
+                    provide: HTTP_INTERCEPTORS,
+                    useClass: ErrorInterceptor
```

Comment:
is error interceptor part of notifications module?
can notifications be used without error interceptor?
can error interceptor be used without notifications?
answer this questions and you will understand your mistake